docs/quarantine.md

```diff
@@ -66,7 +67,8 @@ consecutive executions with the fix pass the test can be removed from the batch.
 #### Quarantined test owners
 
 Each quarantined test must have a team owner. The PR will add the text
-`[sig-{compute,network,storage,operator}]` to each test's description.
+`[sig-{compute,network,storage,operator}]` to each test's description and
+the proper label decorator.
```

Comment:
@dhiller look here for the https://github.com/kubevirt/kubevirt/pull/10488/files/ec0ffdb72189f39af0c571b2f971c8276e09c8c2#r1338586637 answer